language/repl.md

```diff
@@ -0,0 +1,20 @@
+# REPL
+
+REPL stands for "read eval print loop". It's a convenient tool that
+reads a nix-expression and prints the evaluated result.
+
+Nix has such a REPL:
+
+```
+nix-env -iA nixpkgs.nix-repl
```

Comment:
let me suggest "Nix has such a REPL which can be installed with the following command:"